XamarinCommunityToolkit/Views/CameraView/iOS/FormsCameraView.ios.cs

```diff
@@ -51,7 +52,7 @@ public FormsCameraView()
 			};
 
 			mainView.Layer.AddSublayer(previewLayer);
-			RetrieveCameraDevice(CameraOptions.Default);
+			//RetrieveCameraDevice(CameraOptions.Default);
```

Comment:
Should we just remove this so we won't wonder why it was there in the first place? :D

Reply:
yes, I just noticed that!